### DISCUSSION TITLE
Values From Related Module - field not showing up

### DISCUSSION THREAD

**Discussion by TonyM15** [YetiForceCompany/YetiForceCRM#16568] [How to] [answered]
Created "Values From Related Module" field in Sales Orders (value from projects) but this field not showing up in module. Some config?

**Comment by rskrzypczak** [comment, marked as answer]
This type of field, for obvious reasons, is not available in the edit view, but it should be in the detailed view.
If the field is not available in general, check the permissions in the profiles for this field.
